robot_start_node: 0
